### DISCUSSION TITLE
Finacial management, cask and bank account extensions.

### DISCUSSION THREAD

**Discussion by manhtd98** [saleor/saleor#11154] [Q&A] [answered]
### What are you trying to achieve?

I want to store my finacial status, bank account and money transection. How to do it within saleor

### Steps to reproduce the problem

new money management

### What did you expect to happen?

Graphql to get cask and bank account finacial status 

### Environment

Saleor version: 4.0
OS and version: …

**Comment by peelar** [comment]
Hello @manhtd98 👋 

I advised you to follow the issue template, as you created an issue. Now I see that you probably didn't want to report any issues, just ask questions about Saleor. For that reason, I converted it into a discussion. I hope that's alright.

Unfortunately, I still need some more details from you. What is your exact use case? Can you explain what you mean by "financial status" or a "bank account"?

In the meantime, please take a look at our documentation for [Cart and Checkout](https://docs.saleor.io/docs/3.x/developer/checkout).

**Comment by timuric** [comment, marked as answer]
@manhtd98 you can use metadata to store arbitrary account details
